api/rpc/account/account.go

```diff
@@ -215,6 +216,7 @@ func (s *Server) GetUser(ctx context.Context, in *GetUserRequest) (*GetUserReply
 }
 
 // ListUsers implements account.ListUsers
+// nolint : dupl
```

Comment:
I think the default value is ok. If we set the threshold too high, we're probably going to miss actual duplication. We have a lot of duplicates in test code and data storage which makes sense considering the nature of the code.